students/shibata/session3/mailroom.py

```diff
@@ -0,0 +1,150 @@
+#! /usr/bin/env Python
+"""
+It should have a data structure that holds a list of your donors and a history of the amounts they have donated. This structure should be populated at first with at least five donors, with between 1 and 3 donations each.
+
+You can store that data structure in the global namespace.
+The script should prompt the user (you) to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”)
+"""
+
+from time import sleep
+import pprint
+
+if __name__ == '__main__':
+    objFileName = "C:\Files\donate.txt"
+    strData = ""
```

Comment:
non need to create any empty one.

and prefixing with "str" is not really pythonic. -- plus it ends up being a two-element list -- so not a string at all!

plain old "data" is fine.